probes/securityPolicyContainsText/impl.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// nolint:stylecheck
+package securityPolicyContainsText
+
+import (
+	"embed"
+	"fmt"
+
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/utils"
+)
+
+//go:embed *.yml
+var fs embed.FS
+
+var probe = "securityPolicyContainsText"
+
+func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
+	var findings []finding.Finding
+	policies := raw.SecurityPolicyResults.PolicyFiles
+	for i := range policies {
+		policy := policies[i]
```

Comment:
note: this doesn't avoid the copy. is the policy struct so big that it recommends avoiding the copy?

Reply:
changed to `policy := &policies[i]` which was my original intent.. :(